xbmc/guilib/GUIDialog.h

```diff
@@ -45,6 +45,7 @@ class CGUIDialog :
   void Render() override;
 
   void Open(const std::string &param = "");
+  void Open(bool bProcessRenderLoop, const std::string &param = "");
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types